nvm.sh

```diff
@@ -108,16 +108,16 @@ nvm_download() {
   elif nvm_has "wget"; then
     # Emulate curl with wget
     ARGS=$(nvm_echo "$@" | command sed -e 's/--progress-bar /--progress=bar /' \
-                            -e 's/--compressed //' \
-                            -e 's/--fail //' \
-                            -e 's/-L //' \
-                            -e 's/-I /--server-response /' \
-                            -e 's/-s /-q /' \
-                            -e 's/-sS /-nv /' \
-                            -e 's/-o /-O /' \
-                            -e 's/-C - /-c /')
+      -e 's/--compressed //' \
+      -e 's/--fail //' \
+      -e 's/-L //' \
+      -e 's/-I /--server-response /' \
+      -e 's/-s /-q /' \
+      -e 's/-sS /-nv /' \
+      -e 's/-o /-O /' \
+      -e 's/-C - /-c /')
     # shellcheck disable=SC2086
-    eval wget $ARGS
+    eval wget ${ARGS}
```

Comment:
changing this line scares me; can you convince me this is identical in every environment?

Reply:
You mean supported?

Reply:
I reverted this part until I confirmed it's 100% the same.